lib/src/view/game/game_body.dart

```diff
@@ -523,9 +526,59 @@ class _GameBottomBar extends ConsumerWidget {
                   orElse: () => null,
                 ),
               ),
-            // TODO replace this space with chat button
-            const SizedBox(
-              width: 44.0,
+            Stack(
+              children: [
+                BottomBarButton(
+                  label: context.l10n.chat,
+                  shortLabel: context.l10n.chat,
+                  onTap: () {
+                    pushPlatformRoute(
+                      context,
+                      builder: (BuildContext context) {
+                        return MessageScreen(game: gameState.game);
+                      },
+                    );
+                  },
+                  icon: CupertinoIcons.chat_bubble,
+                ),
+                ref.watch(chatControllerProvider).when(
```

Comment:
I prefer to have all `ref.watch` declarations at the beginning of the `build()` method so you know at first glance what widget listen to what.